### DISCUSSION TITLE
Wiki

### DISCUSSION THREAD

**Discussion by gucio321** [AllenDang/giu#664] [Ideas]
Hi!
If you find any bug or missing documentation in [our wiki](https://github.com/AllenDang/giu/wiki) let us know here!

**Comment by greenthepear** [comment]
Since there's a [windows cross-compilation section for MacOS](https://github.com/AllenDang/giu/wiki#build-windows-executable-from-macos) and it took me a hot second to cross-compile [Anaxim](https://github.com/greenthepear/Anaxim) for windows from **Linux** instead - it could be worth to include a section for linux too, even thought it's essentially just a simpler version of the macos procedure. Here's my proposal:
``````markdown
### Build Windows executable from Linux

1. Install mingw64 gcc and g++
    - Fedora/Red Hat: `sudo dnf install mingw64-gcc-c++`
    - Debian/Ubuntu: `sudo apt install gcc-mingw-w64 g++-mingw-w64`
2. Run `go build` with proper environment variables:
```bash
GOOS=windows GOARCH=amd64 CGO_ENABLED=1 CC="x86_64-w64-mingw32-gcc" CXX="x86_64-w64-mingw32-g++" go build -o YourExeName.exe
```
You can also use zig instead of mingw:
```bash
GOOS=windows GOARCH=amd64 CGO_ENABLED=1 CC="zig cc -target x86_64-windows" CXX="zig c++ -target x86_64-windows" go build -o YourExeName.exe
```
``````

**Reply by gucio321** [reply]
wiki updated. Thank you!

**Reply by gucio321** [reply]
btw, I have workflow like this - it deploys giu apps. It may be worth putting somewhere: https://github.com/gucio321/tic-tac-go/blob/master/.github/workflows/release-package.yml
